packages/client/utils/getTokenFromSSO.ts

```diff
@@ -18,27 +24,19 @@ const getTokenFromSSO = (url: string) => {
       if (event.origin !== window.location.origin) return
 
       const params = new URLSearchParams(popup.location.search)
-      const userId = params.get('userId')
+      const userId = params.get('userId')!
```

Comment:
-1 I don't like asserting here just for analytics. If we get this wrong, we'll throw an error and something breaks